### PR TITLE
Make playground visible for mobile devices

### PR DESCRIPTION
**Pull request by Lotes**

### CODE REVIEW

**Comment by Lotes** [comment]
I reused an old branch. Maybe that is why the history is so long O.o

File: hugo/layouts/partials/langium-mobile-menu.html [hunk 1]
```diff
@@ -3,21 +3,18 @@
 x-transition:enter-start="opacity-0 scale-95" x-transition:enter-end="opacity-100 scale-100"
 x-transition:leave="transition ease-in duration-100 transform"
 x-transition:leave-start="opacity-100 scale-100" x-transition:leave-end="opacity-0 scale-95"
-class="absolute top-0 inset-x-0 p-2 transform origin-top-right md:hidden">
+class="absolute inset-x-0 top-0 p-2 origin-top-right transform md:hidden">
```
Comment (montymxb) [inline hugo/layouts/partials/langium-mobile-menu.html:6]: Is there a particular reason why the css classes have been reordered?

**Review by montymxb — COMMENTED** [review]
Question about the css classes, beyond that the single navbar change looks good. If the classes don't need to be switched around, it would be nice to remove the extra changes in this PR. Although if they're needed, we can add them in too.

File: hugo/layouts/partials/langium-mobile-menu.html (hunk 1 above)
Comment (Lotes) [reply hugo/layouts/partials/langium-mobile-menu.html:6]: Good point! No it happens automatically on save...

Comment (Lotes) [reply hugo/layouts/partials/langium-mobile-menu.html]: I disabled my Headwind extension. It was once required for another customer.

**Review by montymxb — APPROVED** [review]
Approved 🧙‍♂️

**Pull request merged**